assemblies/pmr-libraries/src/main/descriptors/assembly.xml

```diff
@@ -158,6 +158,7 @@
         <include>org.eclipse.jetty:jetty-server</include>
         <include>org.eclipse.jetty:jetty-servlet</include>
         <include>org.eclipse.jetty:jetty-util</include>
+        <include>org.eclipse.jetty:jetty-xml</include>
```

Comment:
@smmribeiro did you double check with the @pentaho/moseisley team if this is needed to be included?

Reply:
No, I did not, @graimundo.

There’re 9 jetty components’ dependencies declared (<big-data-plugin>\assemblies\pmr-libraries\pom.xml) and of those, only jetty-xml is not included in the assembly; **however**, all 9 exist on the corresponding lib folder on an installation (<PENTAHO_FOLDER>\design-tools\data-integration\lib).
As so, I’ve added the component to the assembly.xml
Did I assume wrongly? Do you think I should check with them?

On a side note: there're a few other inconsistencies between pom.xml and assembly.xml (3 dependencies that are not assembled, and 10 jars that are assembled and aren't dependencies).

Reply:
> Did I assume wrongly? Do you think I should check with them?

I don't know if you assumed wrongly, hence we should ask @pentaho/moseisley as they have the domain knowledge here. Please double check with them. Also bring up the remaining inconsistencies as we might take the chance and clean those up as well.

Reply:
Resolving as I'm talking to @pentaho/moseisley.